File: dayTwo/python_phase_gate/level 4/function_level_four.py




def add(number):

    return number + number
    
def even(number):
    if(number % 2 == 0):
        print("Even number")   
    else:
    
        print("old number")
        

def square(number):

    return number * number
    
        
        
def celsius(number):
    
    fahrenheit = (number * 1.8) + 32;
    
    return fahrenheit; 
          


def largest(number_one, number_two, number_three):
    largest = number_one
    if(number_two > largest):
        largest = number_two 
    if(number_three > largest):
        largest = number_three        
   
    return largest  
    
    
def simple_interest(timeRate, time):
    
                   
    return ((timeRate / 100) * time) / 100;

  
           
        
print(add(2)) 
even(2) 
print(square(2))
print(celsius(5))
print(largest(2,5,7))
print(simple_interest(5,20))

   

File: dayTwo/JavaPhaseGate/level 4/FunctionLevelFour.java


public class LevelFour{

    public static int adds(int number){
    
      return number + number;

    
    } 
    
    public static void even(int number){
        if(number % 2 == 0){
            System.out.println("even number");}
        
        else{
           System.out.println("old number");
        
        }
        
      }  
        
    public static int square(int number){
    
      return number * number;

    
    }   
    
    public static double celsius(int number){
    
        double fahrenheit = (number * 1.8) + 32;
    
      return fahrenheit;

    
    }  
    
    
     public static int primeNumber(int number){
       
           if (number == 1);
                return false;
          for(int count = 1; count<= (number/2) count++){
                int counter = 0;
              for(int divisor = 1; divisor<=count;  divisor++){
              
            if(number % count){
                return false;
              
              } 
                
          
                }      
             if (counter == 2){
                    return true;
             
             }
            
            }

    
    }  


     public static int largest(int numberOne, int numberTwo, int numberThree){
    
        int largest = numberOne;
            if(numberTwo > largest){
                largest = numberTwo ;
            }
            
            if(numberThree > largest){
                largest = numberThree ;
            
            }            
      return largest;

    }  
    
    
     public static double simpleInterest(int timeRate, int time){
    
                   
      return ((timeRate/100) * time) / 100;

    }  
     
    
    
                        
        
    
      
    
    
     public static void main(String... args){
      
    System.out.println(adds(2));
    even(2);
    System.out.println(square(5));
    System.out.println(celsius(5));
    System.out.println(largest(5,6,1));
    System.out.println(simpleInterest(15,60));
    System.out.println(primeNumber(15));


    }

}
